bin/kafka-server-start.sh

```diff
@@ -16,7 +16,7 @@
 
 if [ $# -lt 1 ];
 then
-	echo "USAGE: $0 [-daemon] server.properties [--override property=value]*"
+	echo "USAGE: $0 [-daemon] [--soft-file-limit number] server.properties [--override property=value]*"
```

Comment:
Just curious, if it would make sense to have it as part of kafka-env.sh, as suggested in KAFKA-1566?